src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -26,11 +26,13 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
+    addMockMentor(response); // TODO: remove by production
     List<UserEntity> mentorEntities = ofy().load().type(UserEntity.class).list();
     ArrayList<UserData> mentors = new ArrayList<UserData>();
     for (UserEntity entity : mentorEntities) {
       if (entity.isMentor()) {
-        mentors.add(new UserData(entity));
+        UserData mentorData = new UserData(entity);
+        mentors.add(mentorData);
```

Comment:
Rather than storing `UserData(entity)` in a variable then appending it to `mentors`, skip the step where you store it in a variable. This reduces 2 lines of code into 1.
```suggestion
        mentors.add(new UserData(entity));
```